scripts/setup_host.sh

```diff
@@ -10,6 +10,10 @@ set -eE
 #---------      Global variable     -------------------
 reboot_required=0
 QEMU_REL="qemu-7.0.0"
+
+#Directory to keep versions of qemu which can be reused instead of downloading again
+QEMU_CACHE_DIR="$HOME/.civ/qemu"
```

Comment:
suggest to use $HOME/Downloads/ or /tmp/

Reply:
I wanted to keep it hidden so that user does not get confused with some new directory and delete it unknowingly. Also, we can make use of .civ for other purposes in future. /tmp would get deleted after restart, so probably not suitable for this purpose. Many other tools use similar directory structure inside $HOME, like .npm, .cache, .m2 . So, I would recommend to continue using this existing path.

Reply:
then $HOME/.cache seems a better place